src/plays/keeper/Keeper.jsx

```diff
@@ -1,23 +1,23 @@
 import { getPlayById } from 'meta/play-meta-util';
 import PlayHeader from 'common/playlists/PlayHeader';
-import './Keeper.css'
+import './Keeper.css';
 import Header from './Header';
 import CreateArea from './CreateArea';
 import Note from './Notes';
-import useLocalStorage from './Hooks/useLocalStorage';
+import useLocalStorage from '../../common/hooks/useLocalStorage';
```

Comment:
The import should work like this too?

import useLocalStorage from 'common/hooks/useLocalStorage';

Reply:
I tried that path at first. But it gives the following error message: 
Module not found: Error: You attempted to import /common/hooks/useLocalStorage which falls outside of the project src/ directory. Relative imports outside of src/ are not supported.
You can either move it inside src/, or add a symlink to it from project's node_modules/.

Reply:
Interesting: See this usage. We are doing already

https://github.com/reactplay/react-play/blob/34dbcb1b6d8035b8813e92dfdcce6f5b83215cca/src/plays/react-todo-app/ReactTodoApp.jsx#L3

Reply:
Should you add hooks here?

https://github.com/reactplay/react-play/blob/main/src/common/index.js